Specs/Core/BoundingSphereSpec.js

```diff
@@ -913,5 +913,33 @@ defineSuite([
         expect(computedVolume).toEqualEpsilon(expectedVolume, CesiumMath.EPSILON6);
     });
 
+    var scratchCartesian3 = new Cartesian3();
+    it('clampToBounds given input value outside of sphere', function() {
+        var sphere = new BoundingSphere(Cartesian3.ZERO, 3.0);
+        var testLocation = new Cartesian3(5, 5, 5);
+        var result = sphere.clampToBounds(testLocation, scratchCartesian3);
+
+        expect(result).not.toEqual(testLocation);
+
+        var result2 = sphere.clampToBounds(result, scratchCartesian3);
```

Comment:
I don't think this is a very precise test. We should be able to figure out what the result is after clamping (`Cartesian3(3, 3,3)`, correct? So we should check that it is returning that value.

Reply:
Same with the other bounding tests.